driver/src/main/scala/za/co/absa/hyperdrive/driver/drivers/CommandLineIngestionDriver.scala

```diff
@@ -52,10 +53,11 @@ object CommandLineIngestionDriver extends IngestionDriver {
   }
 
   private def setOrThrow(setting: String, configuration: Configuration): Unit = {
-    val settingKeyValue = setting.trim.split("=", 2)
-    if (settingKeyValue.length != 2 || settingKeyValue.exists(_.isEmpty)) {
+    if(!setting.contains(PropertyDelimiter)) {
       throw new IllegalArgumentException(s"Invalid setting format: $setting")
+    } else {
+      val settingKeyValue = setting.split(PropertyDelimiter, 2)
+      configuration.setProperty(settingKeyValue(0).trim, settingKeyValue(1).trim)
```

Comment:
This is bit unsafe. You can use try and then make sure they are both defined, and catch problems with undefined.

Reply:
Hmm, do you have an example when it could throw? Imo, they will always be defined because `setting` always contains the `PropertyDelimiter` if it is in this branch

Reply:
ok